portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,9 +27,7 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private static ArrayList<String> comments = new ArrayList<>
-      (Arrays.asList("Hi there!", "Have a good day!", 
-      "I can't think of any more messages to write"));
+  private static ArrayList<String> comments = new ArrayList<>();
```

Comment:
Why is this a static variable? Does it need to be? Was this part of the tutorial? Curious.

Reply:
I made comments static so that you only have to allocation memory to it once/ initialize it once. I guess it may not be necessary because we're only going to need one instance of DataServlet?

Reply:
Yup the Java servlet innerworks creates on instance of DataServlet that stays running while the servlet is running. So it doesn't matter if its static or not but maybe makes more sense as a reader if its not so they don't wonder why its static haha.